### PR TITLE
Minor improvements to jsdocs for BoundingSphere and Matrix3

### PR DESCRIPTION
**Pull request by kainino0x**

### CODE REVIEW

File: Source/Core/Matrix3.js [hunk 1]
```diff
@@ -201,7 +201,7 @@ define([
      *
      * @param {Number[]} values The column-major order array.
      * @param {Matrix3} [result] The object in which the result will be stored, if undefined a new instance will be created.
-     * @returns The modified result parameter, or a new Matrix3 instance if one was not provided.
+     * @returns {Matrix} The modified result parameter, or a new Matrix3 instance if one was not provided.
```
Comment (hpinkos) [inline Source/Core/Matrix3.js]: `Matrix` -> `Matrix3`

File: Source/Core/Matrix3.js [hunk 2]
```diff
@@ -219,7 +219,7 @@ define([
      *
      * @param {Number[]} values The row-major order array.
      * @param {Matrix3} [result] The object in which the result will be stored, if undefined a new instance will be created.
-     * @returns The modified result parameter, or a new Matrix3 instance if one was not provided.
+     * @returns {Matrix} The modified result parameter, or a new Matrix3 instance if one was not provided.
```
Comment (hpinkos) [inline Source/Core/Matrix3.js]: `Matrix3`

**Comment by kainino0x** [comment]
Oops, thanks for catching that. Fixed.

**Comment by hpinkos** [comment]
@kainino0x great! thanks!

**Pull request merged**